### DISCUSSION TITLE
Designing Scenarios on GCAM

### DISCUSSION THREAD

**Discussion by Craven-Pirate** [JGCRI/gcam-core#310] [Modifying GCAM]
I need to customize my scenarios to suit my risk assessment model development. I want to design based on NGFS scenarios. Suppose I want to build a net zero 2050 scenario, wherein my temperature at the end of 2050 will be 1.5 degrees. How to create such a scenario? I initially thought of changing the global temperature variable and running GCAMdata to build / change other dependencies for the input file. However, there is no such variable of global temperature through which I can look at the organic interaction of dependent variables to achieve that goal.

**Comment by DLoughlin** [comment]
GCAM's target finder will iterate over carbon tax trajectories until it hits a specific target such as radiative forcing or change in global mean temperature. This might provide the functionality you are looking for: https://jgcri.github.io/gcam-doc/user-guide.html#target-finder
